From 83d45faafe24bed9c53937d226d500bba6d41a01 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Mon, 22 May 2017 12:43:51 +0500
Subject: [PATCH] CMakeLists: don't use a version suffix

In OE-Core we do a manual package splitting for llvm
because of various reasons and the splitting does
its job using the filenames under libdir. Using a
version suffix in this scenario breaks up things
for packages that depend on LLVM e.g. mesa.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 78e2e016625..605a3551b62 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ if(NOT DEFINED LLVM_VERSION_PATCH)
   set(LLVM_VERSION_PATCH 0)
 endif()
 if(NOT DEFINED LLVM_VERSION_SUFFIX)
-  set(LLVM_VERSION_SUFFIX svn)
+  set(LLVM_VERSION_SUFFIX "")
 endif()
 
 if (POLICY CMP0048)
-- 
2.11.1

